ray_on_golem/server/run.py

```diff
@@ -20,30 +24,56 @@
 
 def parse_sys_args() -> argparse.Namespace:
     parser = argparse.ArgumentParser(description="Ray on Golem's webserver.")
-    parser.add_argument(
+    subparsers = parser.add_subparsers(dest="command")
```

Comment:
my feeling is that the name of the root parser should be more meaningful ... so either `commands` or `commands_parser` if we mean to be hyper correct...

```suggestion
    commands = parser.add_subparsers(dest="command")
```